src/app.js

```diff
@@ -8,4 +8,8 @@ app.use(express.json());
 app.use(cors());
 app.use('/', indexRouter);
 
+app.use((err, req,next,res) => { 
```

Comment:
```suggestion
app.use((err, req, res, next) => { 
```
@rukundo-kevin This is how the parameters should be passed. next should be passed after req.